home3.3.py

```diff
@@ -0,0 +1,28 @@
+"""String formatting.
+
+we have norway text in old style formatting
+re-write the same text as:
+1 string with format() call
+2 f-string
+use linter(https://github.com/wemake-services/wemake-python-styleguide)
+to check your new created python module for possible linter errors
+try to run code from pycharm/command line
+
+norway_text = 'Automatisering akselererer %syeblikket da roboter vil erobre planeten v%sr. (%s)' % ('ø', 'å', 'Æ')
```

Comment:
These both lines should be out of docstring text.
This is code ( aka initial data)